content/pages/privacy.mdx

```diff
@@ -0,0 +1,206 @@
+
+* use this site, including (without limitation) when you:
+* create a user account;
+* add information to your user profile;
+* purchase any products and/or services through this site;
+* register for access to premium content or request certain premium features; or
+* complete an online contact form to contact us;
+* provide information to us by telephone or through marketing or competition application forms; or
+* send us an email or other communication.
+
+#### IP addresses
+
+This site may also collect Internet Protocol (IP) addresses. IP addresses are assigned to computers on the internet to uniquely identify them within the global network. SSW collects and manages IP addresses as part of the service of providing internet session management and for security purposes. SSW may also collect and use web log, computer and connection information for security purposes and to help prevent and detect any misuse of, or fraudulent activities involving, this site.
+
+### Cookies
+
+This site uses "cookies" to help personalise your online experience. A cookie is a text file or a packet of information that is placed on your hard disk by a Web page server to identify and interact more effectively with your computer. There are two types of cookies that may be used at this site: a persistent cookie and a session cookie. A persistent cookie is entered by your Web browser into the "Cookies" folder on your computer and remains in that folder after you close your browser, and may be used by your browser on subsequent visits to this site. A session cookie is held temporarily in your computer’s memory and disappears after you close your browser or shut down your computer. Cookies cannot be used to run programs. Cookies are uniquely assigned to you, and can only be read by a Web server in the domain that issued the cookie to you. In some cases, cookies may collect and store personal information about you. SSW extends the same privacy protection to your personal information, whether gathered via cookies or from other sources.
+You can configure your internet browser to accept all cookies, reject all cookies or notify you when a cookie is sent. Please refer to your internet browser’s instructions to learn more about these functions. Most Web browsers automatically accept cookies, but you can usually modify your browser settings to decline cookies if you prefer. If you choose to decline cookies, you may not be able to fully experience the interactive features of this site.
+
+#### Why SSW use cookies
+
+This site uses cookies in order to:
+
+* remember your preferences for using this site;
+* manage the signup process when you create an account with us;
+* recognise you as logged in while you remain so. This avoids your having to log in again every time you visit a new page;
+* facilitate e-commerce transactions, to ensure that your order is remembered betSSWen pages during the checkout process;
```

Comment:
As per my conversation with Cameron, the text on line 74 is a typo. The text should be changed: 
from: 
        'is remembered betSSWen pages' 
to: 
        'is remembered between pages'

Reply:
Fixed